code/modules/mob/language/outsider.dm

```diff
@@ -77,17 +77,17 @@
 /datum/language/xenocommon
 	name = "Xenolingua" //CHOMPedit
 	colour = "alien"
-	desc = "The common tongue of both the xenomorphs and the Genaprawns." //CHOMPedit
+	desc = "The common tongue of both the Xenomorphs and the Genaprawns." //CHOMPedit
```

Comment:
If "xenomorph" is going to be a species name, it's not a proper noun. Lowercase.